apps/addons/models.py

```diff
@@ -508,12 +512,15 @@ def initialize_addon_from_upload(cls, data, is_listed=True):
         if not locale_is_set:
             addon.default_locale = to_language(trans_real.get_language())
 
+        addon.save()
+        if old_guid_addon:
+            old_guid_addon.update(guid='guid-reused-by-pk%s' % addon.pk)
```

Comment:
You didn't. I'm just inflicting some noise. Again. Will shut up for now, until I wake up